hw1/02-palindrome.js

```diff
@@ -1,3 +1,23 @@
 const elem = document.querySelector('input');
-
+const resultText = document.getElementById('resultText');
 elem.addEventListener('input', handleInput);
+
+function handleInput(){
+    let input = parseInt(elem.value);
+    if(input >= 0){
+        let reverseInput = parseInt(String(elem.value).split('').reverse().join(''));
+        if( input == reverseInput){
+            resultText.style.color = 'green';
+            resultText.innerHTML = 'Yes. This is a palindrome!';
+        }else{
+            resultText.style.color = 'red';
+            resultText.innerHTML = 'No. Try again.'
+        }
+    }else if(input<0){
+        resultText.style.color = 'red';
+        resultText.innerHTML = 'Enter a positive number!';
+    }else{
+        resultText.style.color = 'black';
+        resultText.innerHTML = 'Please enter a number first!';
+    }
```

Comment:
There is a lot of redundancy in using `resultText.style.color =` and `resultText.innerHTML = `. I think it's better to create a variable for the new color and HTML color then just assign those to `resultText` instead

Reply:
I have reduced the redundancy by using a function to update the color class and text. Thank you for your review!